spec/openapi.yaml

```diff
@@ -1293,14 +1295,11 @@ paths:
         - ```subject``` The subject field is used to denote subject of the MMS
         message and has a maximum size of 64 characters long. Specifying a
         subject is optional.
-
-        - ```source_number_type``` If a source number is specified, the type of
-        source number may also be
-          specified. This is recommended when using a source address type that is not an internationally
-          formatted number, available options are ```INTERNATIONAL```, ```ALPHANUMERIC``` or ```SHORTCODE```. Specifying a
-          source number type is only valid when the ```source_number``` parameter is specified and is optional.
-          If a source number is specified and no source number type is specified, the source number type will be
-          inferred from the source number, however this may be inaccurate.
+        
+        - ```source_number``` A source number can be specified for the message,
```

Comment:
Actually, can you move this to be right after the `source_number_type`? The idea is to move `source_number_type` to be above `source_number` not to swap them around.

Reply:
Okay will update